docs/user/openstack/install_upi.md

```diff
@@ -392,26 +393,55 @@ You are free to choose any storage you want. For example:
 
 * Swift Object Storage
   Create the `<container_name>` container and upload the `bootstrap.ign` file:
+
   ```sh
   $ swift upload <container_name> bootstrap.ign
   ```
+
   Make the container accessible:
+
   ```sh
   $ swift post <container_name> --read-acl ".r:*,.rlistings"
   ```
-  Get the `StorageURL` from the output:
+
+  Get the `storage_url` from the output:
+
   ```sh
   $ swift stat -v
   ```
+
+* Glance image service
+
+  Create the `<image_name>` image and upload the `bootstrap.ign` file:
+
+  ```sh
+  $ glance image-create --name <image_name> --disk-format=raw --container-format=bare --file bootstrap.ign
```

Comment:
Can we do this with `openstack image create` instead? The only reason we're using `swift` instead of `openstack` above is that you can't set the Swift container ACL with python-openstackclient.

But I think image creation should work and we're using `openstack` everywhere else.

Reply:
done